src/Artsy/Analytics/TrackingContext.tsx

```diff
@@ -0,0 +1,66 @@
+import { track as _track } from "Artsy/Analytics"
+import * as AnalyticsSchema from "Artsy/Analytics/Schema"
+import React from "react"
+
+type Trackables = AnalyticsSchema.Trackables
+
+/**
+ * Used when tracking custom events, not in the schema:
```

Comment:
Why would we have these?

Reply:
Having just played around with the code and finding that the same additional type was passed in multiple places, I definitely don’t think we should do this, it will only introduce drift in the schema. So while I left [the generic in](https://github.com/artsy/reaction/pull/2433/files#diff-dbc9dc55a5bdbaf4e74b5a92ca77fd51R31) in my changes, I think we should not allow this.